source/conditional_loops.rst

```diff
@@ -54,7 +54,7 @@ a ``while`` loop, and some new turtle functions:
 * ``turtle.towards(0,0)`` - The angle to get back to origin
 * ``turtle.setheading(angle)`` - Directly sets the turtle direction
 
-You could try playing with a turtle in the interpreter and using these 
+You could try playing with a turtle in the interpreter using these 
```

Comment:
I think adding a comma between interpreter and using, will produce the best grammar here.